### ISSUE TITLE
install: cannot create regular file issue when installing manpages

### ISSUE THREAD

**Issue by Earnestly** [eXeC64/imv#160]
The `Makefile` uses a combination of `mkdir -p` and `install -m` (possibly due to macos lacking `install -D`) when attempting to install the manual pages:

```make
	mkdir -p $(DESTDIR)$(MANPREFIX)
	$(INSTALL_MAN) doc/imv.1 $(DESTDIR)$(MANPREFIX)/man1/imv.1
```

However the call to `mkdir` only creates the `MANPREFIX` and not the proceeding `/man1` such that when `install -m` is used, it cannot create `man1/imv.1` (without `-D`).

A local workaround on linux systems (or anything with GNU install) would be to redefine `INSTALL_MAN` as `install -Dm0644`.

Another appoach might be to use GNU standard variables such as `man1dir`, `man2dir`, `man3dir`, etc. variables instead (e.g. `man2dir = $mandir/man2` where `mandir = $datarootdir/man`). https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

**Comment by eXeC64** [comment]
The `-D` flag is avoided to retain compatibility with BSD's `install` command. For that same reason, I'd like to avoid GNU-specific variables.

To keep it simple, It seems to me like `mkdir` command ought to have `/man1` at the end of it.

**Comment by Earnestly** [comment]
Yup. that would fix it

**Comment by jubalh** [comment]
That fixes it for me too.

**Comment by eXeC64** [comment]
Fixed by 5b6618afd747c5396977dcbc06570f81f2a48d06 
v3.1.2 released to include this packaging fix
